index.js

```diff
@@ -7,7 +7,9 @@ const XXH = require('xxhashjs');
 
 const H = XXH.h32(0xABCD) // seed = 0xABCD
 
-const appIndexJs = path.resolve(pwd, 'src/index.js');
+const appIndexes = ['js', 'ts', 'jsx', 'tsx'].map(
+  ext => path.resolve(pwd, `src/index.${ext}`
```

Comment:
I was afraid of the compatible of Template Literal and arrow function, maybe we'll add babel and test in future, Thanks a lot!

Reply:
As far as I can see both of those features are supported by node 6.x